tests/models/test_models.py

```diff
@@ -280,6 +280,25 @@ def test_scvi_sparse(save_path):
     model.differential_expression(groupby="labels", group1="label_1")
 
 
+def test_setting_adata_attr():
+    n_latent = 5
+    adata = synthetic_iid()
+    SCVI.setup_anndata(adata, batch_key="batch")
+    model = SCVI(adata, n_latent=n_latent)
+    model.train(1, train_size=0.5)
+
+    adata2 = synthetic_iid()
+    model.adata = adata2
+    model.get_latent_representation()
```

Comment:
check this is equivalent to doing `model.get_latent_representation(adata2)` and not `model.get_latent_representation(adata)`.